packages/block-library/src/page-list/edit.js

```diff
@@ -76,11 +76,9 @@ export default function PageListEdit( { context, clientId } ) {
 
 			{ hasResolvedPages && totalPages === null && (
```

Comment:
I think that these two conditions will not ever be met.

Reply:
I think this will be true for users with lower capabilities who cannot access the pages. See the original PR - #37486.